downstream/modules/platform/con-editing-inventory-files.adoc

```diff
@@ -6,4 +6,4 @@ You can further configure your {PlatformName} installation by including addition
 These configurations add optional features for managing your {PlatformName}. 
 Add these variables by editing the inventory file using a text editor.
 
-A table of predefined values for inventory file variables can be found in link:{BaseURL}/red_hat_ansible_automation_platform/2.3/html/red_hat_ansible_automation_platform_installation_guide/appendix-inventory-files-vars[Inventory File Variables] in the _{PlatformName} Installation Guide_.
\ No newline at end of file
+A table of predefined values for inventory file variables can be found in link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html/red_hat_ansible_automation_platform_installation_guide/appendix-inventory-files-vars[Inventory file variables] in the _{PlatformName} Installation Guide_.
```

Comment:
{PlatformVers} here is 2.3?
Below it's 2.4

Reply:
I presumed that was an error and anything in the 2.4 doc version should be referencing 2.4 etc. 
Should I change this instance back to 2.3?